balancer-js/src/modules/exits/exits.module.integration.spec.ts

```diff
@@ -0,0 +1,549 @@
+import dotenv from 'dotenv';
```

Comment:
Some of these tests are failing for me locally - do you also see that?

Reply:
They were passing on my end. Sometimes, when I was running them for the first time, they were timing out. I'd need to  run them a second time, which was faster. I believe this is due to some sort of local fork cache.
Let's see how it goes after we merge the fix for the github workflow.
And I'll double check on my end just in case.

Reply:
Strange. It wasn't timing out, it was actually showing a BAL error. Will be interested to see what you see.

Reply:
I just checked and apparently it's the same issue where Tenderly runs simulation agains the current block while the test is running on another blocknumber. This leads to limits being set incorrectly and the test fails.
We can either adapt the test to always run against the current block or adapt the Tenderly helper to receive blocknumber as a parameter.
Since this parameter would be relevant for tests only, I believe we shouldn't add it. I'll go with removing blocknumber from tests for now. Let me know if you think we should go with the other approach.

Reply:
Sorry if you've already made these changes but maybe we should make Tenderly work with a specified blockNumber. I'm just thinking there could be issues where pools have all liquidity removed or similar and then the tests will fail and we'll have to debug, etc. We could make the blockNo an optional argument and default to most recent if not supplied.